axelrod/strategies/finite_state_machines.py

```diff
@@ -148,28 +153,56 @@ class Predator(FSMPlayer):
 
     @init_args
     def __init__(self):
-        transitions = ((0, C, 0, D),
-                       (0, D, 1, D),
-                       (1, C, 2, D),
-                       (1, D, 3, D),
-                       (2, C, 4, C),
-                       (2, D, 3, D),
-                       (3, C, 5, D),
-                       (3, D, 4, C),
-                       (4, C, 2, C),
-                       (4, D, 6, D),
-                       (5, C, 7, D),
-                       (5, D, 3, D),
-                       (6, C, 7, C),
-                       (6, D, 7, D),
-                       (7, C, 8, D),
-                       (7, D, 7, D),
-                       (8, C, 8, D),
-                       (8, D, 6, D))
+        transitions = (
+            (0, C, 0, D),
+            (0, D, 1, D),
+            (1, C, 2, D),
+            (1, D, 3, D),
+            (2, C, 4, C),
+            (2, D, 3, D),
+            (3, C, 5, D),
+            (3, D, 4, C),
+            (4, C, 2, C),
+            (4, D, 6, D),
+            (5, C, 7, D),
+            (5, D, 3, D),
+            (6, C, 7, C),
+            (6, D, 7, D),
+            (7, C, 8, D),
+            (7, D, 7, D),
+            (8, C, 8, D),
+            (8, D, 6, D)
+        )
 
         FSMPlayer.__init__(self, transitions, initial_state=1, initial_action=C)
 
 
+class Pun1(FSMPlayer):
+    """FSM player described in [Ashlock2006]."""
```

Comment:
Could you write this docstrings to match up with the convention (not all of them follow it yet, slow conversion :)). In particular for this it should have (appended to what is already there).

```
Names

    - Pun1 [Aslhlock2006]_
```